CHANGELOG.md

```diff
@@ -66,11 +84,18 @@ be found.
 + Add `--stop-signal` for `service create` and `service update` [#30754](https://github.com/docker/docker/pull/30754)
 + Add `--read-only` for `service create` and `service update` [#30162](https://github.com/docker/docker/pull/30162)
 + Renew the context after communicating with the registry [#31586](https://github.com/docker/docker/pull/31586)
++ (experimental) Add `--tail` and `--since` options to `docker service logs` [#31500](https://github.com/docker/docker/pull/31500)
++ (experimental) Add `--no-task-ids` and `--no-trunc` options to `docker service logs` [#31672](https://github.com/docker/docker/pull/31672)
 
 ### Windows
+
 * Wait for OOBE to prevent crashing during host update [#31054](https://github.com/docker/docker/pull/31054)
 * Block pulling Windows images on non-Windows daemons [#29001](https://github.com/docker/docker/pull/29001)
 
+### Deprecation
```

Comment:
hm, ok, wondering if it's better to have a separate section to more clearly point it out (it's easily overlooked and may be important when upgrading from a previous version)

open to suggestions though